bciers/libs/utils/customTransformErrors.ts

```diff
@@ -6,6 +6,9 @@ const customTransformErrors = (
 ) => {
   return errors.map((error) => {
     if (error?.property) {
+      if (error.message === "must be equal to constant") {
+        error.message = undefined; // this is to handle errors generated by dependencies
+      }
```

Comment:
What was causing this? Why are we setting the message to `undefined`, is this just a workaround to stop an erroneous validation error?

Reply:
Without this, if you click "yes" to parent operators and then hit submit without filling out all the required parent operator fields, then we get "must be a constant" validation error on the yes/no field even though it's filled. I think this is because we're using "oneOf" , so it flags the whole subschema as problematic if any one piece is missing. Same thing happens with the partner operators

Reply:
Makes sense, thanks for filling me in!